YAML/tests/syntax_test_types.yaml

```diff
@@ -133,6 +133,11 @@
    .NaN, ~, .Nan,
 #  ^^^^ meta.number.yaml constant.language.nan.yaml
 #           ^^^^ - constant
+## Note: it's uncertain whether `._1` should be considered valid.
```

Comment:
Highlighting helps with smoother UX while editing though. We don't neeed or shouldn't be too restrictive anyway.

Reply:
In this case I'm not sure, because it makes a difference to see if something is actually (processed as) a number or a piece of text, no?